pkg/api/kptfile/v1/types.go

```diff
@@ -49,6 +49,9 @@ type KptFile struct {
 	// UpstreamLock is a resolved locator for the last fetch of the package.
 	UpstreamLock *UpstreamLock `yaml:"upstreamLock,omitempty" json:"upstreamLock,omitempty"`
 
+	// Origin is a resolved locator for the last pull or push of the package.
+	Origin *Origin `yaml:"origin,omitempty" json:"origin,omitempty"`
```

Comment:
This is a good addition IMO.

Does `kpt pkg push` remove `origin` information before pushing to the remote artifact? 
I see that `kpt pkg get` removes the `origin` section by default, since the user intent is to consume the package?, please confirm.

I tried the e2e workflows and they look good. Thank you.

Reply:
Yep! It does: at a minimum the digest value would be immediately wrong as soon as it was pushed. But even worse the image can be moved around so every part of the origin data could be wrong by the time you pull it from a different location.